README.md

```diff
@@ -1,348 +1,143 @@
+particularly good at surviving configuration changes, weathering the storm 
+as Android destroys your portrait Activity and Views and replaces them 
+with landscape doppelgangers. 
+
+Mortar can similarly make [Dagger][dagger] ObjectGraphs visible as system
+services, or [Dagger2][dagger2] Components, or any other service that might be
+useful to your apps. Or not &mdash; these services are completely decoupled.
+
+Everything is managed by [MortarScope][scope] singletons, typically
+backing the top level Application and Activity contexts. You can also spawn
+your own shorter lived scopes to manage transient sessions, like the state of
+an object being built by a set of wizard screens.
+
+These nested scopes can shadow the services provided by higher level scopes.
+For example, a [Dagger extension graph][ogplus] specific to your wizard session
+can cover the one normally available, transparently to the wizard Views &mdash;
+they can make calls like `ObjectGraphService.inject(getContext(), this)` without
+considering which graph will do the injection.
 
 ## The Big Picture
 
-A Mortar app has only a handful of Activity classes, ideally a single one. It
-does not have Fragments or Loaders. Instead its UI is built primarily of plain
-old [Views][view], which use [Dagger][dagger] to `@Inject` whatever services
-they need.
-
-Typically such a view is a thin thing that delegates all of its interesting
-work to an injected controller of type [Presenter][presenter]. A Presenter
-simplifies life by surviving configuration changes, and has just enough access
-to the Activity lifecycle to be restored after process death.
-
-As your app grows you probably want to divide it up into discrete parts, both
-to retain your sanity and to be miserly with the scarce resources available on
-a mobile device. You can divide it into a tree of [MortarScopes][scope]. Each
-scope is defined by a [Blueprint][blueprint] that can serve as a single point
-of reference for what that scope does, and in particular can define objects
-visible only to the scope and its children.
-
-Maybe a scope is associated with a particular view, or maybe it isn't. For
-example, an app might have a top level global scope that allows the injection
-of fundamental services; a child of that which manages objects that require an
-authenticated user; children of them for each Activity; and children of the
-activity scopes for their various screens and regions.
-
-### Lifecycle
-
-A MortarScope provides a simple life cycle: it can tell interested parties
-when they enter and exit it if they implement the [Scoped][scoped] interface.
-Scopes created at the Activity level or below can also manage instances of the
-[Bundler][bundler] interface, which give access to the host Activity's
-persistence Bundle. And that's the entire lifecycle that Mortar apps need to
-deal with:
+A Mortar app will typically have a singleton MortarScope instance
```

Comment:
Also, isn't this just a description of how the service provider stuff works?  Why present it as application architecture?